discourse-doctor.rb

```diff
@@ -1,18 +1,26 @@
 #!/usr/bin/env ruby
+# coding: utf-8
 
 Dir.chdir("/") do
   require 'bundler/inline'
   require 'net/http'
   require 'net/smtp'
+  require 'pathname'
 
   gemfile(true, ui: false) do
     source 'https://rubygems.org'
 
     gem 'dnsruby'
     gem 'paint'
+    gem 'sys-filesystem' # for checking available disk space
```

Comment:
To have that working correctly you are assuming that `df`

- outputs exactly one line as a header
- prints the available blocks in the fourth row
- always uses the same blocksize

Additionally it's harder to read (you have to know df, tail, awk).